application/src/main/webapp/vue-app/components/ExoChatContact.vue

```diff
@@ -173,6 +174,15 @@ export default {
     document.removeEventListener(chatConstants.EVENT_RECONNECTED, this.setOnline);
   },
   methods: {
+    getSpaceURI() {
+      const spaceId = this.name.toLowerCase().split(' ').join('_');
+      getSpaceByPrettyName(this.name).then((space) => {
+        if (space && space.identity) {
+          this.spaceGroupUri= space.groupId.replace(/\//g, ':');
+        }
+      });
+      return `${eXo.env.portal.context}/g/${this.spaceGroupUri}/${spaceId}`;
```

Comment:
```suggestion
      return getSpaceByPrettyName(this.name).then((space) => {
        if (space && space.identity) {
          this.spaceGroupUri= space.groupId.replace(/\//g, ':');
          return `${eXo.env.portal.context}/g/${this.spaceGroupUri}/${spaceId}`;
        }
      });
```

Reply:
it will return the value before the promise is fulfilled and change `spaceGroupUri` value.